integrations/cypress/cypress/support/utils/config.ts

```diff
@@ -10,35 +10,36 @@ interface Config {
   clientID: string;
   domain: string;
   sdk: Auth0SDKs
+  auth0RunningPort: number;
+  auth0SimulatorPort: number;
 }
 
 export function getConfig(): Config {
-  let sessionCookieName = Cypress.env('auth0SessionCookieName') ?? 'appSession';
-  let cookieSecret = Cypress.env('auth0CookieSecret');
-  let audience = Cypress.env('audience');
-  let connection = Cypress.env('connection') ?? 'Username-Password-Authentication';
-  let scope = Cypress.env('scope') ?? 'openid profile email offline_access';
-  let clientSecret = Cypress.env('auth0ClientSecret');
-  let clientID = Cypress.env('clientID') ?? 'YOUR_AUTH0_CLIENT_ID';
-  let domain = Cypress.env('domain') ?? 'localhost:4400';
-  let sdk = (Cypress.env('AUTH0_SDK') ?? 'auth0_react') as Auth0SDKs;
-
+  let auth0RunningPort = Cypress.env('AUTH0_RUNNING_PORT') ?? 4400;
   return {
-    sessionCookieName,
-    cookieSecret,
-    audience,
-    connection,
-    scope,
-    clientSecret,
-    clientID,
-    domain,
-    sdk
+    auth0RunningPort,
+    auth0SimulatorPort: Cypress.env('AUTH0_SIMULATOR_PORT') ?? 4000,
```

Comment:
Do we need this, you can set this using `PORT=4000` without another environment variable.

Reply:
PORT is quite generic and it was initially confusing for me that this was not actual Auth0 server but the GraphQL interface so I chose to make it distinct from each-other.

Reply:
After re-reading this I understand it now, the port can already be changed so this extra `env` variable is not needed. I ended up removing it altogether

Reply:
I managed to get your tests passing in `./examples/create-react-app`.  There were a few gotchas and especially running `--headless`.

Any `cy.log` in a promise eg. `cy.log(`successfully logged in with token ${JSON.stringify(token)}`);` will cause the test to fail with 

>  CypressError: The following error originated from your test code, not from Cypress. It was caused by an unhandled promise rejection.

  > Cypress detected that you returned a promise from a command while also invoking one or more cy commands in that promise.

The command that returned the promise was:

  > `cy.visit()`

The cy command you invoked inside the promise was:

  > `cy.log()`

Reply:
Any chance this PR can be merged, and not yet publish a new package, and then we continue from the other PR? The other PR already include fixes for these and I fixed the packaging in that one. It has become difficult to track what is what a new issues keep arising. 

Or I merge both into 1 but then the PR is even more huge

Reply:
> Any chance this PR can be merged, and not yet publish a new package, and then we continue from the other PR? The other PR already include fixes for these and I fixed the packaging in that one. It has become difficult to track what is what a new issues keep arising.
> 
> Or I merge both into 1 but then the PR is even more huge

I personally would not merge this because the `--headless` tests are not working because the calls to `cy.log` make them fail. 

Also the example tests in the create-react-app project do not work and have not been updated.  I quickly got a [hacky branch](https://github.com/thefrontside/simulacrum/tree/pc/create-react-app-fixes) together to get them passing.

That branch was a total hack

That is my shout anyway, but I do not work for frontside anymore, so maybe @taras @jbolda or @cowboyd have a different opinion.  I'm also a bit pushed for time at the moment

I really appreciate the work you are doing but I don't like merging stuff that is not finished but the others might differ.

Reply:
Fair points, I underestimated the number of changes that were needed and at this point it's almost a complete rewrite. Ideally several smaller PR's would have been in order. 

Thanks for the branch, I will take a look and use it to fix this PR!